tracks/python/exercises/leap/mentoring.md

```diff
@@ -1,3 +1,24 @@
+### Most common first attempt
+
+This is the most common solution by beginners to the leap year problem.
+Nudge the student towards the more concise solution by letting them know
+they can replace the 3 IF and 4 return statements with zero IF statements and a single return.
+This is done by using the logical operators OR & AND.
```

Comment:
Use backticks and real operator names, maybe?

Also, is it true that this is the most common solution by beginners?  If that is the case, I don't have a problem with stating so.  If it hasn't been confirmed by actually counting, then I would feel uncomfortable claiming that, or presenting it, as fact.

Reply:
> Use backticks and real operator names, maybe?

I'm sorry, I don't understand what you mean? 


I have seen two students write virtually the same code as presented in this pull request, In fact, on my first attempt with the leap year problem, I arrived at the same code. I don't have the statistics to back up the claim, but given that the leap year problem focuses on control flow, I would assume that beginners would gravitate to solving this problem using multiple IF statements. 

That being said, I understand where you are coming from and I'm happy to reword the mentor guide to claim _it is one of the most common solutions_ rather than the definitive claim it _is the most common_.

Reply:
For the backticks and real operator names, I am talking about doing this:

```
they can replace the 3 `if` and 4 `return` statements with zero `if` and a single `return`.
```

The reason is in part the consistency.  The if and the return words are mixed case, the "IF" and the "return" not being in the same style, so it appears there may be two different kinds of things. (And maybe it is appropriate.)  Hopefully that makes sense.  I mean to say that `if` is a real thing, as is `return` and being code, you can use the backtick to indicate this.

I might say "a common solution may be" and that would be indicative.

We could more simply say "if this solution is presented" without making the judgement of "beginner" or "many" or "most".  This may come from those beginning in the language, but well versed in a different programming language.  We don't care so much.  But we can give suggestions for this kind of solution.